README.md

```diff
@@ -1,6 +1,67 @@
+[![Check branch is releasable](https://github.com/empathyco/x/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/empathyco/x/actions/workflows/build.yml)
+[![Created by](https://img.shields.io/badge/Created%20by-Empathy.co-green)](https://www.empathy.co)
 
-The Search eXperience interface components
+Search inevitably is key when people look for specific products in an online shop. Search is the door to the entire product catalogue, so having an outstanding, well-structured, and optimised UI is a must.
+
+Interface X is the solution. This renewed search experience interface layer, available for web and mobile, renders the magic of search and delights your shoppers with all experiences in one: user, search, navigation, and product discovery.
+
+Transform your search UI, simplify product discovery, and configure the shopper experience with cutting-edge technology based on Vue.js and out-of-the-box components.
+
+
+## Ecosystem
+
+Interface X is formed by [some packages](./.github/CONTRIBUTING.md#interface-x-and-packages) that are being moved to this mono-repo during the 2021 Spring-Summer. 
+We encourage you to watch this project as it will be updated every week. 
+
+As you can see there are a few packages but most of them are minor dependencies, we recommend to focus on two of them:
+
+* [@empathyco/x-components](https://github.com/empathyco/x/tree/main/packages/components) - This is the core package of this project. 
+These are standalone and configurable building blocks that allow you to quickly construct the search UI for your shop. 
+You can create a smooth, personalised search and discovery experience, while significantly minimising development time. (To be moved here soon) 
+* [@empathyco/x-archetype](https://github.com/empathyco/x/tree/main/packages/archetype) - This package is the Empathy vision of the 
+perfect X-Components mixing. A project showing the power of the components ready to connect to any Search API with customizable layout through Design Tokens. 
+This is a perfect example to know how to use the X-Components. (To be moved here soon) 
+
+## Roadmap
+
+2021 will be the year to consolidate Interface X, the roadmap includes this highlighted milestone:
+
+* Build the Archetype configurations and No-Code design system.
+* URL Management to set X-Components state based on URL parameters
+* Create a tagging system to track user interactions
+* Search box power ups as animated suggestions or forbidden character set to prevent code injection
+* Publish a extensive documentation covering functional and technical sides.
+* To be an open source project! 
+
+## Fast, flexible, and straightforward 
+
+* **Easy-to-add interface layer**. Installation and setup of the interface is simple: just add a few lines of code.
+* **Use it anywhere.** Search interface layer that is easy to integrate into any website, with X Components ready to use in your Vue.js and React projects.  
```

Comment:
` **Use it anywhere.** Search interface layer is easy to integrate into any website...` I'm not sure about this one but the `that` word sounds weird.

Reply:
It would have more sense saying:

`A search interface layer that is...`